frontend/src/components/admin/ApprovedLanguagesTable.tsx

```diff
@@ -221,122 +169,50 @@ const ApprovedLanguagesTable = ({
     closeApproveLanguageModal(false);
   };
 
-  const tableBody = approvedLanguages.map(
-    (approvedLanguage: ApprovedLanguage, index: number) => (
-      <Tr
-        borderBottom={
-          index === approvedLanguages.length - 1 ? "1px solid gray" : ""
-        }
-        key={`${approvedLanguage.language}/${approvedLanguage.role}`}
-      >
-        <Td>{convertLanguageTitleCase(approvedLanguage.language)}</Td>
-        <Td>{approvedLanguage.role}</Td>
-        <Td colSpan={4} align="center">
-          <Slider
-            isDisabled={!isAdmin}
-            defaultValue={approvedLanguage.level}
-            min={1}
-            max={4}
-            step={1}
-            marginLeft="20px"
-            size="lg"
-            width="79%"
-            onChange={(val: number) =>
-              onSliderValueChange(
-                approvedLanguage.role === "Translator",
-                approvedLanguage.language,
-                val,
-                approvedLanguage.level,
-              )
-            }
-          >
-            <SliderTrack height="6px">
-              <SliderFilledTrack bg="blue.500" />
-            </SliderTrack>
-            <SliderThumb boxShadow="0px 0px 3px black" />
-            {[1, 2, 3, 4].map((val) => (
-              <SliderMark
-                key={val}
-                value={val}
-                height="6px"
-                width="4px"
-                marginTop="-3px"
-                // #64A1CD is light blue, #ECF1F4 is light gray
-                bg={approvedLanguage.sliderValue > val ? "#64A1CD" : "#ECF1F4"}
-              />
-            ))}
-          </Slider>
-        </Td>
-        {!isAdmin && (
-          <Tooltip
-            hasArrow
-            label="Currently at maximum language level."
-            isDisabled={approvedLanguage.level !== 4}
-          >
-            <Td>
-              <Button
-                aria-label="Level up approval language and level"
-                background="white"
-                size="sm"
-                width="103px"
-                variant="blueOutline"
-                isDisabled={approvedLanguage.level === 4}
-              >
-                Level Up
-              </Button>
-            </Td>
-          </Tooltip>
-        )}
-      </Tr>
-    ),
-  );
+  const closeNewTestModal = () => {
+    setAlertNewTest(false);
+  };
+
+  const closeLevelUpModal = () => {
+    setTestLevel(0);
+    setTestLanguage("");
+    setConfirmLevelUp(false);
+    setAlertNewTest(true);
```

Comment:
this causes the second modal to open even when the first one is closed (cancelled)